.github/workflows/ci-cd.yml

```
@@ -587,18 +587,214 @@
+            else
+              echo "⏳ Deployment not ready yet, waiting 10 seconds..."
+              sleep 10
+            fi
+            
+            if [ $i -eq 30 ]; then
+              echo "❌ Deployment not ready after 5 minutes, but continuing with Lighthouse test"
+            fi
+          done
+
+      - name: Run Lighthouse CI on deployed site
+        run: |
```

actionlint (1.7.7)

property "deploy-production" is not defined in object type {}

(expression)